app/models/course/material/folder.rb

```diff
@@ -10,6 +10,12 @@ class Course::Material::Folder < ActiveRecord::Base
   # TODO: Remove this after schema_validations #21 was closed
   validates :name, uniqueness: { case_sensitive: false, scope: :parent_id }, if: :parent
 
+  def files=(files)
```

Comment:
Then in the form I also need to write `f.files_attributes`, this looks weird and makes me a bit uncomfortable ><  But OK